lib/ansible/modules/cloud/azure/azure_rm_cosmosdbaccount_facts.py

```diff
@@ -0,0 +1,418 @@
+                    returned: always
+                    type: str
+                    sample: cosmosdbaccounxx-eastus
+                document_endpoint:
+                    description:
+                        - Document endpoint.
+                    returned: always
+                    type: str
+                    sample: https://cosmosdbaccounxx-eastus.documents.azure.com:443/
+                provisioning_state:
+                    description:
+                        - Provisioning state.
+                    returned: always
+                    type: str
+                    sample: Succeeded
+        database_account_offer_type:
+            description:
+                - Offer type.
+            returned: always
+            type: str
+            sample: Standard
+        ip_range_filter:
+            description:
+                - Enable IP range filter.
+            returned: always
+            type: str
+            sample: 10.10.10.10
+        is_virtual_network_filter_enabled:
+            description:
+                - Enable virtual network filter.
+            returned: always
+            type: bool
+            sample: true
+        enable_automatic_failover:
+            description:
+                - Enable automatic failover.
+            returned: always
+            type: bool
+            sample: true
+        enable_cassandra:
+            description:
+                - Enable Cassandra
+            returned: always
+            type: bool
+            sample: true
+        enable_table:
+            description:
+                - Enable Table.
+            returned: always
+            type: bool
+            sample: true
+        enable_gremlin:
+            description:
+                - Enable Gremlin.
+            returned: always
+            type: bool
+            sample: true
+        virtual_network_rules:
+            description:
+                - List of Virtual Network ACL rules configured for the Cosmos DB account.
+            type: list
+            contains:
+                subnet:
+                    description:
+                        - Resource id of a subnet.
+                    type: str
+                    sample: /subscriptions/subid/resourceGroups/testrg/providers/Microsoft.Network/virtualNetworks/testvnet/subnets/testsubnet1
+                ignore_missing_vnet_service_endpoint:
+                    description:
+                        - Create Cosmos DB account without existing virtual network service endpoint.
+                    type: bool
+        enable_multiple_write_locations:
+            description:
+                - Enable multiple write locations.
+            returned: always
+            type: bool
+            sample: true
+        document_endpoint:
+            description:
+                - Document endpoint.
+            returned: always
+            type: str
+            sample: Succeeded
+        provisioning_state:
```

Comment:
pls add a flag to return connection string or key. otherwise user will not able to use the cosmosdb

Reply:
yeah, that may be useful :-)

Reply:
done